src/sempy_labs/migration/_migrate_tables_columns_to_semantic_model.py

```
@@ -111,7 +111,7 @@ def migrate_tables_columns_to_semantic_model(
 
                         try:
                             tom.model.Tables[tName]
-                        except:
+                        except Exception:
```

is an exception thrown if the table does not exist? can you check if the table exists first, then catching the exception. again this is masking all other errors.

---

updated this so it doesn't need a try/except clause